README.md

```diff
@@ -121,3 +125,7 @@ python manage.py loaddata dumpdata.json
 ### Deployment
 
 The app is set to automatically push from the master branch to the staging area (https://ldmw-app-staging.herokuapp.com/). When we're happy this is working, we will push manually to the production site.
+
+### CMS
+
+The documentation for the CMS (Creating pages, tags etc) can be found at https://github.com/LDMW/cms#cms
```

Comment:
markdown link: "can be found at [https://github.com/LDMW/cms#cms](https://github.com/LDMW/cms#cms)" ?